Apps/scripts/getNightly.js

```diff
@@ -0,0 +1,21 @@
+var https = require('https');
+var fs = require('fs');
+
+function download(filename, url) {
+  var file = fs.createWriteStream(filename);
+  var request = https.get(url, function(response) {
+    response.pipe(file);
+  });
+}
+
+console.log('Downloading babylon.js nightly');
+download('node_modules/babylonjs/babylon.max.js', 'https://preview.babylonjs.com/babylon.max.js');
+download('node_modules/babylonjs/babylon.max.js.map', 'https://preview.babylonjs.com/babylon.max.js.map');
+download('node_modules/babylonjs-materials/babylonjs.materials.js', 'https://preview.babylonjs.com/materialsLibrary/babylonjs.materials.js');
+download('node_modules/babylonjs-materials/babylonjs.materials.js.map', 'https://preview.babylonjs.com/materialsLibrary/babylonjs.materials.js.map');
+download('node_modules/babylonjs-loaders/babylonjs.loaders.js', 'https://preview.babylonjs.com/loaders/babylonjs.loaders.js');
+download('node_modules/babylonjs-loaders/babylonjs.loaders.js.map', 'https://preview.babylonjs.com/loaders/babylonjs.loaders.js.map');
+download('node_modules/babylonjs-gui/babylon.gui.js', 'https://preview.babylonjs.com/gui/babylon.gui.js');
+download('node_modules/babylonjs-gui/babylon.gui.js.map', 'https://preview.babylonjs.com/gui/babylon.gui.js.map');
+download('node_modules/chai/chai.js', 'https://unpkg.com/chai/chai.js');
+download('node_modules/mocha/mocha.js', 'https://unpkg.com/mocha/mocha.js');
```

Comment:
I know this is just moving code, but why are we doing this? Doesn't `npm install` already bring down the right files?

Reply:
version on `unpkg.com` are more recent than npm. It looks like Darragh did it because of an issue with one of these NPM packages : https://github.com/BabylonJS/BabylonNative/pull/897

Reply:
CI is OK when download is removed. I guess it was a bug with a specific NPM package that's been fixed since.

Reply:
Interesting. We don't do this locally. Was this a CI only issue?

Reply:
Darragh must have been the only one testing unittests locally. Or the mocha/chai issue has been fixed not long after this has been put in the CI.

Reply:
I have run them a few times since this was added. Never seen any issues. Anyways, sounds like we can remove this now.

Reply:
It's been removed.